service/controller/resource/monitoring/prometheus/resource_test.go

```diff
@@ -23,13 +23,13 @@ func TestPrometheus(t *testing.T) {
 		Installation:      "test-installation",
 		Pipeline:          "testing",
 		Provider:          "provider",
-		PrometheusVersion: "v2.27.1",
 		Region:            "onprem",
 		Registry:          "quay.io",
 		StorageSize:       "50Gi",
 		RetentionDuration: "2w",
 		RetentionSize:     "45Gi",
 		RemoteWriteURL:    "http://grafana/api/prom/push",
+		Version:           "v2.27.1",
```

Comment:
What's the reasoning behind this one? Asking cause just `Version` is not self-explanatory and could be confusing.

Reply:
I only renamed it for the prometheus resource config and it is still called prometheus version everywhere else. Otherwise, we should theoretically name RemoteWriteURL PrometheusRemoteWriteURL and so on. I think the redundancy is not useful here

Reply:
Oh, because the full path of this thing is `prometheus.Config.Version`, right, my bad, didn't notice that.